DESCRIPTION

```diff
@@ -30,10 +30,12 @@ Authors@R:
              comment = c(ORCID = "0000-0001-9560-6336", Twitter = "@bmwiernik")))
 Maintainer: Dominique Makowski <dom.makowski@gmail.com>
 Description: The 'easystats' is a meta-package that contains a set of
```

Comment:
Either delete "The" or add "package" after "'easystsats'"

Reply:
Added "package" after "'easystsats'"